emmo/utils.py

```diff
@@ -420,7 +424,7 @@ def infer_version(iri, version_iri):
     return version
 
 
-def ontology_annotation(onto, imported=True):
+def annotate_with_ontology(onto, imported=True):
```

Comment:
This functions seems to work well, should it perhaps be added as option to Ontology.save?

Reply:
That could also be a possibility. I original intention was to add it as an option to the ontoconvert tool.